designer/client/src/components/graph/Graph.tsx

```diff
@@ -395,6 +398,7 @@ export class Graph extends React.Component<Props> {
             this.processGraphPaper.hideTools();
             if (isStickyNoteElement(cellView.model)) {
                 if (!this.props.isPristine) return;
+                if (this.props.selectionState.length > 0) this.props.resetSelection(...[]);
```

Comment:
Do we need to spread an empty array here?
```suggestion
                if (this.props.selectionState.length > 0) this.props.resetSelection();
```

Reply:
Yeah, you are right, thanks :)